### ISSUE TITLE
Text Background Color Issue.

### ISSUE THREAD

**Issue by fauzi16** [tinymce/tinymce#3263]
Hy,

I got several issues regarding with text formatting when testing our product, below is steps to reproduce : 
1. type the text in editor
2. select the text
3. change font size of the text
4. click on bold/underline/italic/strikethrougth
5. give the background color.

I have recorded this.
![text-formatting issue](https://cloud.githubusercontent.com/assets/20159853/19299823/d408f732-9080-11e6-84e8-5163363ae2fa.gif)

Please tinyMCE fix this..

Thanks.

**Comment by alecpl** [comment]
The HTML code produced is:
```
<p><span style="color: #ff6600; background-color: #33cccc;"><em><span style="font-size: 36pt;">ITALIC</span></em></span></p>
```
but should be:
```
<p><span style="color: #ff6600; background-color: #33cccc; font-size: 36pt;"><em>ITALIC</em></span></p>
```
then it would work.

**Comment by fyrkant** [comment]
Fixed in yesterdays 4.5.3 release.
